### PR TITLE
Cleaning time~

### PR DESCRIPTION
**Pull request by tigercat2000**
Deleting old files and reorganizing somewhat.

If I deleted any useful files, or you know of more disabled files that are safe to delete, please tell me.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Looks good to me.

:+1:

File: code/game/mecha/equipment/tools/unused_tools.dm [hunk 1]
```diff
@@ -1,157 +0,0 @@
...
-			target.gib()
-			target.client.gib()
-			target.client.mom.monkeyize()
-			target.client.mom.gib()
-			for(var/mob/M in range(target, 1000))
-				M.gib()
-			explosion(target.loc,100000,100000,100000)
-			usr.gib()
-			world.Reboot()
-			return 1
```
Comment (Crazylemon64) [inline code/game/mecha/equipment/tools/unused_tools.dm]: nuuuu

File: code/game/socket_talk.dm [hunk 2]
```diff
@@ -1,26 +0,0 @@
...
-		receive_raw()
-			if(enabled)
-				return call("DLLSocket.so","recv_message")()
-		send_log(var/log, var/message)
-			return send_raw("type=log&log=[log]&message=[message]")
-		send_keepalive()
-			return send_raw("type=keepalive")
-
-
-var/global/datum/socket_talk/socket_talk
```
Comment (Crazylemon64) [inline code/game/socket_talk.dm]: Does this mean we can get rid of the DLLsocket top-level directory, too?

File: code/game/mecha/equipment/tools/unused_tools.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/game/mecha/equipment/tools/unused_tools.dm]: Wow. Lol.

Comment (Spacemanspark) [inline code/game/mecha/equipment/tools/unused_tools.dm]: ...That was a thing?
@tigercat2000, we need this finished.

Comment (Fox-McCloud) [inline code/game/mecha/equipment/tools/unused_tools.dm]: It literally gibs everyone on the entire server, spawns a size 100,000 explosion, gibs the user, then reboots the server. Lol

Comment (Spacemanspark) [inline code/game/mecha/equipment/tools/unused_tools.dm]: I know.
That's why we need it.

**Pull request merged**